src/java.base/windows/classes/sun/nio/fs/WindowsFileCopy.java

```diff
@@ -174,7 +177,14 @@ static void copy(final WindowsPath source,
 
         // Use CopyFileEx if the file is not a directory or junction
         if (!sourceAttrs.isDirectory() && !sourceAttrs.isDirectoryLink()) {
-            final int flags = (!followLinks) ? COPY_FILE_COPY_SYMLINK : 0;
+            long size = 0;
+            try {
+                size = Files.size(source);
```

Comment:
I thought you were going to change this to use sourceAttrs.size() but it looks like the call to Files.size has come back in this version.

Reply:
Fixed in 406656e254004113f80bb36351db6662ad01a8f9.